tests/src/test/scala/org/apache/openwhisk/core/controller/test/WhiskAuthHelpers.scala

```diff
@@ -25,7 +25,11 @@ object WhiskAuthHelpers {
     WhiskAuth(s, Set(WhiskNamespace(Namespace(EntityName(s.asString), k.uuid), k)))
   }
 
-  def newIdentity(s: Subject = Subject(), uuid: UUID = UUID(), k: GenericAuthKey = BasicAuthenticationAuthKey()) = {
-    Identity(s, Namespace(EntityName(s.asString), uuid), k, Privilege.ALL)
```

Comment:
this method was buggy when using a BasicAuthenticationAuthKey and default values because the uuid will not match.